README.mkd

```diff
@@ -229,6 +230,15 @@ registers and then save `ra` and the appropriate number of registers from
 epilogue. They restore the saved registers, deallocate the stack space for the
 register, and then perform a return through the restored value of `ra`.
 
+`__riscv_restore_tailcall_<N>` are additional entry points used when the
+epilogue of the calling function ended in a tail-call. Unlike
```

Comment:
ended -> end?

Reply:
"end" is not grammatically correct.
"ended" (or "ends") is.

Reply:
Sorry, I meant "ends". The point is that it seemed more natural to me to talk about a function that presently exists and ends in a tail call. But it's just a minor nit and I'll leave the grammar and style to the native speakers :)